cli/flox-rust-sdk/src/models/manifest/typed.rs

```diff
@@ -946,4 +981,23 @@ pub(super) mod test {
         assert_eq!(filtered.inner().len(), 1, "{:?}", filtered);
         assert!(filtered.inner().contains_key("postgres"));
     }
+
+    #[test]
+    fn parses_include_section_manifest() {
+        let manifest = indoc! {r#"
+            version = 1
+
+            [include]
+            environments = [
+                { dir = "../foo", name = "bar" },
+            ]
+        "#};
+        let parsed = toml_edit::de::from_str::<Manifest>(manifest).unwrap();
+
+        assert_eq!(parsed.include.environments.len(), 1);
+        let included = parsed.include.environments[0].clone();
+        let IncludeDescriptor::Local { dir, name } = included;
+        assert_eq!(dir, PathBuf::from("../foo"));
+        assert_eq!(name.as_str(), "bar");
+    }
```

Comment:
nonblocking: would probably be good to add a docs-composition section and modify manifest.toml.md. We can also come back to that later

Reply:
I'll create tickets for those